.ruby-gemset

```diff
@@ -0,0 +1 @@
+usds-home
```

Comment:
Commenting here because it won't let me comment on the .DS_Store.

We should add .DS_Store to the .gitignore